mk/compile.mk

```diff
@@ -146,3 +148,76 @@ $(foreach f, $(gen-srcs), $(eval $(call \
 	process_srcs,$(out-dir)/$(f),$(out-dir)/$(base-prefix)$$(basename $f).o)))
 
 $(objs): $(conf-file)
+
+define gen-defines-file
+# c-filename in $1
+# h-filename in $2
+# s-filename in $3
```

Comment:
Yes, but it's very convenient like this. It's use at many places, I'm not sure it's possible to just put it in a local variable with a short name. Perhaps I should rewrite this without a macro instead?